packages/synapse-react-client/src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -27,26 +26,29 @@
       SRC.$bootstrap-4-danger-text-color
     );
   }
+
+  // PORTALS-2597, SWC-6413: all buttons should be rounded
+  button {
+    border-radius: SRC.$button-border-radius;
```

Comment:
Could perhaps remove if the scss variable above works
```suggestion
```

Reply:
This block handles the case where a MUI button has an ancestor with the `bootstrap-4-backport` class -- the MUI button-radius / text-transform are overridden by `button` styling in bootstrap's `_reboot.scss`. See Date Posted button in [DiscussionThread](https://sage-bionetworks.github.io/synapse-web-monorepo/?path=/story/synapse-discussionthread--discussion-thread-demo). 

Since the MUI buttons don't have a `btn` class, the variable override isn't applied here, so we style the `button` element directly. We could refactor components that contain MUI buttons to remove the `bootstrap-4-backport` class, but that would be a larger lift.